### DISCUSSION TITLE
Ruska propaganda

### DISCUSSION THREAD

**Discussion by PolishRoboDog** [FiltersHeroes/KAD#2070] [Pomysły]
### Link bezpośredni 
`https://pl.sputniknews.com`
### Dodatkowe informacje mogące mieć znaczenie
Ruska rządowa propaganda z fake newsami skierowanymi m.in. przeciwko Polsce.

Zgłoszenie opublikowane anonimowo przez użytkownika.

**Comment by admin** [comment]
Myślę, że podobny problem do propagandy Białoruskiej #1988  (#1983).

**Comment by admin** [comment]
W tym wypadku jednak pewnie całość by się kwalifikowała do zablokowania, aczkolwiek jednakże jest tu parę opcji językowych, więc pewnie bardziej kwalifikowałoby się to do jakiejś globalnej listy.
Niby zablokowali już to polscy operatorzy, ale ci co korzystają z innych DNS-ów nadal mogą mieć dostęp.

```
https://www.wirtualnemedia.pl/artykul/serwisy-i-kanaly-youtube-owe-sputnika-i-russia-today-zablokowane-w-polsce
https://forsal.pl/swiat/unia-europejska/artykuly/8369860,czescy-operatorzy-blokada-6-stron-internetowych-fake-newsy.html
https://www.computerworld.pl/news/Koniec-Russia-Today-i-Sputnika-w-Europie,436706.html
```
